src/ray/rpc/client_call.h

```diff
@@ -3,6 +3,7 @@
 
 #include <grpcpp/grpcpp.h>
 #include <boost/asio.hpp>
+#include "absl/synchronization/mutex.h"
```

Comment:
Is there an advantage to using `absl` rather than the standard classes? Seems to introduce an extra dependency but I'm not sure what it's gaining in exchange (I'm not familiar with it).

Reply:
Yes, the compiler can check the `GUARDED_BY` annotations and make sure the necessary locks are held when variable are accessed.